13.03

lkulbacki opened the pull request:
Finished section 13 with extras

lkulbacki committed:
Change move definition from internal JS into data-attribute

lkulbacki committed:
Move global variables into params and elements objects

lkulbacki committed:
Add modals and displaying final result in modal

lkulbacki committed:
Add progress table to results

lkulbacki committed:
Merge branch '13.03_upgrade_kamien_nozyce_papier' into 13.03

lkulbacki committed:
Update package.json configuration for windows compatibility

lkulbacki committed:
Add "New game" modal

stepek commented on 08_podstawy_js_gra_kamien_nozyce_papier/.gitignore:
```diff
@@ -73,4 +73,5 @@ typings/
-sass/*.css
\ No newline at end of file
+sass/*.css
+css/*.css
```
dlaczego dodajesz to do gitignore?

lkulbacki replied:
Folder CSS: autoprefixer na Macu i Windowsie inaczej zapisywał wyjściowe pliki CSS, w związku z czym z każdym commitem powstawała nic nie wnosząca różnica. Pliki CSS zawsze można wygenerować taskiem buildującym. Ale OK, przywrócę śledzenie CSS bo faktycznie w tym projekcie nie ma tworzenia paczki jako takiej.

Folder SASS: pliki CSS musiały mi się pojawić przypadkowo w trakcie konfiguracji tasków i je roboczo zakomentowałem, wyrzucę tą linikję.

stepek reviewed (commented):
Proszę dodaj poprawki z rozmowy.

stepek commented on 08_podstawy_js_gra_kamien_nozyce_papier/js/script.js:
```diff
@@ -39,77 +46,138 @@ var getResult = function (playerSymbol, computerSymbol) {
+    }
+    if (params.gamesLimit !== 'undefined') {
+        params.gameOn = true;
+        params.gamesPlayed = 0;
```
zerowanie można wyjąć do osobnej funkcji

lkulbacki replied:
Przeniesione.

lkulbacki committed:
Add scaffold of params passing between IIFY functions

lkulbacki committed:
Remove unneccessary comments in .gitignore

https://github.com/lkulbacki/kodilla-web-dev-plus/pull/13#discussion_r236379815

lkulbacki committed:
Make modal content scrollable

lkulbacki committed:
Refactor: extract modal support functions from game logic script

lkulbacki committed:
Cleanup code and add comments

lkulbacki commented:
@stepek wyekstraktowałem funkcjonalność modali i owinąłem script.js w IIFY; zrobiłem też mały refactor i okomentowałem kod; prośba o weryfikację

lkulbacki committed:
Refactor: resetting parameters moved to separate function
